sycl/test-e2e/lit.cfg.py

```diff
@@ -281,6 +281,8 @@
 if "ext_oneapi_hip:gpu" in config.sycl_devices and config.hip_platform == "AMD":
     config.available_features.add('hip_amd')
     arch_flag = '-Xsycl-target-backend=amdgcn-amd-amdhsa --offload-arch=' + config.amd_arch
+    if "gfx90a" in config.sycl_devices:
+        config.available_features.add("gpu-amd-gfx90a")
```

Comment:
How does this work? I thought that `sycl_devices` has the format of `ONEAPI_DEVICE_SELECTOR` and I doubt `gfx90a` is part of it.

Reply:
Thanks @aelovikov-intel 
You are right, I modified it.